### ISSUE TITLE
Linux compatibility

### ISSUE THREAD

**Issue by darkdragon-001** [Rookiestyle/KeePassOTP#27]
## Overview
All versions newer than 16.1 (when Google Authenticator support was added) don't work on my Ubuntu machine any more. There is an error displayed, that a newer version of the framework is required. I am using `Mono JIT compiler version 6.8.0.105 (Debian 6.8.0.105+dfsg-3`. There is an overview of supported .NET features in mono [here](https://www.mono-project.com/docs/about-mono/compatibility/).

## Context
OS: Ubuntu 20.10
KeePass Version: 2.45
Plugin Version: everything > 16.1

**Comment by Rookiestyle** [comment]
It took me a bit but I finally found it.

KeePassOTP required .NET framework >= 4.5 since the very beginning, whereas Mono reports 4.0.3 being installed although actually higher versions can be used.
This should have made the error message being shown for all versions as well. 

Before adding Google Athenticator import, there was no need to rely on NuGet packages and I used my own script to create the plgx. Due to an error in the script this .NET framework requirement was not added to the plgx.

I reworked the plugin and now .NET framework 4.0 is sufficient.
